app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -278,8 +278,11 @@ public LeakTool getLeakTool() {
     UiModeManager uiModeManager = (UiModeManager) getSystemService(Context.UI_MODE_SERVICE);
     aptoideApplicationAnalytics = new AptoideApplicationAnalytics(analyticsManager);
 
+    DelegatingWorkerFactory factory = new DelegatingWorkerFactory();
```

Comment:
Seems unnecesary given that we're doing the delegation inside the factory already. I guess we can either remove it, or create another factory for the new worker.